src/com/jester/model/Model.java

```diff
@@ -30,6 +30,10 @@ public class Model {
     ResultSet rs = null;
     
     
+    public Model() {
+        this.connect = new EstablishConnection(); 

```

Comment:
This is when Model is instantiated on other class files, it will try to get a connection immediately